test/journal/update_journal_entry_test.js

```diff
@@ -105,6 +106,26 @@ describe("Journal", function () {
                 expect(response.body.mood).to.equal("");
             });
         });
+        //moodEmoji
+        it("rejects a blank mood Emoji", function () {
+            return updatePatientEntry({}, {
+                moodEmoji : ""
+            }).then(function (response) {
+                expect(response).to.be.an.api.error(400, "invalid_emoji");
+                expect(response.body.moodEmoji).to.equal("");
```

Comment:
See the comment on line 109 of `create_new_entry_test.js`: API errors don't include record data in their response, so `response.body.moodEmoji` isn't included. That's why this test is failing -- you'll want to remove line 115. You can then reformat it into a one-line statement like the examples in the other comments.